docs/concepts/trading-on-vega/positions-margin.md

```diff
@@ -0,0 +1,133 @@
+
+When marking to market, the protocol takes the current market price and recalculates traders' margin requirements based on how their position is affected by price moves.
+
+If the market price goes up, a trader that holds a long position receives money in their margin account – equal to the underlying's change in value – from a trader that holds a short position, and conversely if the value goes down, the holder of the short position recieves money from the holder of the long position.
+
+For a futures market created on Vega, the mark-to-market price is calculated every time the price moves, and is based on the last traded price. This is in contrast to traditional futures markets, for which marking to market occurs once per day. One exception is when the market settles at expiry, at which point the mark to market price comes from the data source's final settlement price.
+
+Settlement instructions are generated based on the change in market value of the open positions of a party. When the mark price changes, the network calculates settlement cash flows for each party, and the process is repeated each time the mark price changes until the maturity date for the market is reached.
+
+Because the margin for a market is calculated dynamically based on the market conditions, the mark price also has an effect on how much collateral is set aside for margin.
+
+:::note Read more
+[Mark to market settlement](./settlement#mark-to-market-settlement)
+:::
+
+## Margin
+Margin is the amount of collateral required to keep your position open. It can change depending on how your position is impacted by your own actions and market movement. The margin calculations ensure a trader does not enter a trade that will immediately need to be closed out.
+
+The calculation for a new position, and the amount deducted from collateral to cover margin, is based on all of a trader's open orders. A trader will need enough margin to keep a position open, whether it goes for or against the trader.
+
+The margin set aside for a given position is dictated by the specifics of the position and its exposure. For example, if a trader has a long position on a market, and then reduces the position - their margin requirement doesn't get larger.
+
+:::note Go deeper
+**[Automated cross margining](https://vega.xyz/papers/vega-protocol-whitepaper.pdf#page21)** - Section 6 of the protocol whitepaper.
+:::
+
+## Cross-margining
+Vega's margining system provides automated cross margining. Cross margining, which means gains on one market can be released and used as margin on another, is supported between all markets that use the same settlement asset.
+
+To more closely control how much is risked on a position, it's possible to replicate the effects of isolated margin by using one party (public key) per market.
```

Comment:
I've introduced some repletion with a previous suggestion but think this is okay.